ui/components/MesheryFilters/FiltersGrid.js

```diff
@@ -92,6 +108,19 @@ function FiltersGrid({ filters=[],handleDeploy, handleUndeploy, handleSubmit, se
 
       </Grid>
       }
+      {!selectedFilter.show && filters.length === 0 &&
+        <Paper className={classes.noFilterPaper}>
+          <div className={classes.noFilterContainer}>
+            <Typography align="center" color="textSecondary" className={classes.noFilterText}>
+            No Filters Found
+            </Typography>
+            <div>
+              <UploadImport aria-label="URL upload button" handleUpload={urlUploadHandler} handleImport={uploadHandler} configuration="Filter"  />
```

Comment:
@harkiratsm This might be a bit confusing, the param says `handleImport` but the value is `uploadHandler`. How about if we update it to something which can be easily read?

Reply:
@adithyaakrishna 
```handleUpload``` is changed to ```handleUrlUpload``` & ```handleImport``` to ```handleUpload```.